src/models/Podcast.js

```diff
@@ -0,0 +1,11 @@
+import fs from "fs"
+
+const podcastsPath = "podcasts.json"
+
+const podcastsJSON = () => {
```

Comment:
this `podcastsJSON` method should not be provided/external to the class: the source code for `fetch-and-express-with-async-await` has a great example of the pattern we're using